jardin/model.py

```diff
@@ -437,17 +437,16 @@ def find(self, id, **kwargs):
 
     @classmethod
     def db_adapter(self, role='replica', db_name=None):
-        if not hasattr(self, '_db_adapter'):
-            self._db_adapter = {}
+        if not hasattr(self, '_db_metadata'):
+            self._db_metadata = {}
         db_name = db_name or self.db_names.get(role)
         key = '%s_%s' % (db_name, role)
-        if key not in self._db_adapter:
```

Comment:
No need to cache DatabaseAdapter here - make it be consistent with the way we run raw SQLs (jardin.query()).

However, caching model_metadata is still needed.